test/src/utils/path_utils_test.dart

```diff
@@ -112,4 +112,22 @@ void main() {
       expect(ext, 'conf');
     });
   });
+
+  group('getFileTypeFromMime method', () {
+    test('should return the file type from mimetype', () {
+      final type = getFileTypeFromMime(contentType: 'application/pdf');
+
+      expect(type, 'pdf');
+    });
+  });
+
+  group('getBasenameWithoutExtension method', () {
+    test('should return the basename without extension', () {
+      final basename = getBasenameWithoutExtension(
+        filePath: '/hola/que/tal/file.pdf',
+      );
+
+      expect(basename, 'file');
```

Comment:
Same comment as before. Some test cases: 
- filePath: 'file.pdf'
- filePath: 'file.pdf.txt'
- filePath: 'file' <- no extension on the file name (is it possible to get using IOFile?)